lmi_utils/image_utils/img_pad_to_size.py

```diff
@@ -16,9 +16,7 @@ def pad_image(input_path,output_path,W,H):
         print(f'Input file: {input_file}')
         im=cv2.imread(input_file)
         im_out=pad_array(im,W,H)
-        fname=os.path.split(input_file)[1]
-        fname=os.path.splitext(fname)[0]
-        fname=fname+'_'+str(W)+'x'+str(H)+'.png'
+        fname=os.path.basename(input_file)
```

Comment:
1. this would replace the original image if the input and output folder is same
2. it is beneficial to rename the image using new image sizes,m which gives user some traceability

Reply:
as during the modelling, I often need to rely on the same filename in preprocessing, that where the change comes, given tracibility is achievable through output folder name.
regarding #1, how about abort the action if the output dir is the same as input dir?